### PR TITLE
[#140237] Strong params

### PR DESCRIPTION
**Pull request by meara**
https://pm.tablexi.com/issues/140237

### CODE REVIEW

**Comment by jhanggi** [comment]
@meara Are you using this branch anymore?

**Comment by meara** [comment]
Yeah I'm using it to track what still needs to be fixed.

**Comment by meara** [comment]
@jhanggi this could use a review when you've got a minute.

File: app/controllers/products_common_controller.rb [hunk 1]
```diff
@@ -161,8 +161,8 @@ def resource_params
                                                       :schedule_id, :control_mechanism, :reserve_interval,
                                                       :min_reserve_mins, :max_reserve_mins, :min_cancel_hours,
                                                       :auto_cancel_mins, :lock_window, :cutoff_hours,
-                                                      :auto_logout, :auto_logout_mins,
-                                                      relay_attributes: [:ip, :port, :username, :password, :type, :instrument_id])
+                                                      relay_attributes: [:ip, :port, :username, :password, :type,
```
[the remark this answers is not in the document]
Comment (meara) [reply app/controllers/products_common_controller.rb]: We do now.

**Pull request merged**